ios/RNStartTime.m

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+#import "RNStartTime.h"
+#import <sys/sysctl.h>
+#include <math.h>
+#import <QuartzCore/QuartzCore.h>
+
+@implementation RNStartTime
+
+RCT_EXPORT_MODULE()
+
+static CFTimeInterval processStartTime() {
```

Comment:
Can you add some comment about what this function is doing and why do we need a native function for this?